pipelines/release/release.yaml

```diff
@@ -444,6 +457,23 @@ jobs:
             EOF
             echo "merging with cluster values..."
             spruce merge ./platform/charts/gsp-cluster/values.yaml ./overrides.yaml | tee -a cluster-values/values.yaml
+  - task: pick-platform-version
+    image: concourse-task-toolbox
+    config:
+      platform: linux
+      inputs:
+      - name: full-gsp
```

Comment:
I don't think having a `full-gsp` can be relied upon to get the behaviour you want here.

There is no guarentee that "full-gsp" will return the LATEST master HEAD at `get` time and this would also likely cause the pipeline to not act in an idempotent way if you re-run a failed job.

I think you will have to lose this `full-gsp` resource and determin which of the other multiple input git-resources is the "biggest" maybe by counting the number of commits `git rev-list --count master` ?